content/tutorials/manage-the-rippled-server/build-run-rippled-ubuntu.md

```diff
@@ -79,40 +58,45 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
               ./bootstrap.sh
 
-      5. To invoke Boost.Build to build the separately-compiled Boost libraries, run the following command. Replace `<number of parallel jobs>` with the number of jobs to run in parallel. Choose this value based on the number of CPU cores you want to use for building. This may take about 10 minutes, depending on your hardware specs.
+      5. Set parallel jobs to the half the number of CPUs available on your box, with minimum of 1
+      
+              PROCESSORS=$(expr "`grep processor /proc/cpuinfo | wc -l`" / 2)
+
+      6. To invoke Boost.Build to build the separately-compiled Boost libraries, run the following command.
 
-              ./b2 -j <number of parallel jobs>
+              ./b2 -j $PROCESSORS
 
-      6. Set the environment variable `BOOST_ROOT` to point to the new `boost_1_64_0` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
+      7. Set the environment variable `BOOST_ROOT` to point to the new `boost_1_64_0` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
 
-              export BOOST_ROOT=/home/ubuntu/boost_1_64_0
+              cd ~
+              echo "export BOOST_ROOT=$PWD/boost_1_64_0" >> ~/.profile
 
-      7. Source your updated `.profile` file. For example:
+      8. Source your updated `.profile` file. For example:
 
               source ~/.profile
 
-10. From a working directory, get the `rippled` source code. The `master` branch has the latest released version.
+5. From a working directory, get the `rippled` source code. The `master` branch has the latest released version.
 
         cd ~
         git clone https://github.com/ripple/rippled.git
         cd rippled
         git checkout master
 
-11. Check the commit log to be sure you're compiling the version you intend to. The most recent commit should be signed by a well-known Ripple developer and should set the version number to the latest released version. For example:
+6. Check the commit log to be sure you're compiling the version you intend to. The most recent commit should be signed by a well-known Ripple developer and should set the version number to the latest released version. For example:
 
-        $ git log
+        $ git log -1
 
-        commit f31ca2860fb5f045b618aa05d1e76c7e2e9494ec (HEAD, tag: 1.0.0, origin/release, origin/master, master)
-        Author: Nikolaos D. Bougalis <nikb@bougalis.net>
-        Date:   Fri May 11 10:29:41 2018 -0700
+        commit 8429dd67e60ba360da591bfa905b58a35638fda1
```

Comment:
Gonna change this again when 1.1.0 comes out (very soon), but it's fine for now.